### PR TITLE
Admin user reports

### PR DESCRIPTION
**Pull request by AxelBogos**
1. Added a user reports page
2. Ability for admins to filter users
3. Ability for admins to see all outstanding accounts
4. Ability for admins to activate/deactivate accounts

### CODE REVIEW

File: admin_user_reports.php [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        <tr>
+            <form method="post">
+                <td style="padding-bottom: 1em;">
+                    <input type="submit" value="See All Outstanding Accounts" name="seeOutstanding" style="min-width: 170px">
+                </td>
+            </form>
+        </tr>
+    </table>
+    <div>
+        <span class="help-block"><?php echo $deactivate_err; ?></span>
```
Comment (TaillessSheep) [inline admin_user_reports.php:211]: defined variable: deactivate_err

File: admin_user_reports.php [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM oyc353_1.`1User`
+                        WHERE balance<'" . 0 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            //Default table. All users
+            else{
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM oyc353_1.`1User`ORDER BY isEmployer DESC";
```
Comment (TaillessSheep) [inline admin_user_reports.php]: maybe remove the _oyc353_1._ if it is not necessary

File: admin_user_reports.php [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+                        WHERE premiumOpt='" . $category ."'
+                        ORDER BY isEmployer DESC";
+                }
+                $result = mysqli_query($db,$sql);
+            }
+            //See outstanding balance accounts
+            elseif(isset($_POST['seeOutstanding']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM oyc353_1.`1User`
```
Comment (TaillessSheep) [inline admin_user_reports.php]: maybe remove the _oyc353_1._ if it is not necessary

File: admin_user_reports.php [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            // Search by premium option
+            elseif(isset($_POST['filterPremium']))
+            {
+                $category = $_POST['accountCategory'];
+                if($category == 'all' || empty($category))
+                {
+                    $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM oyc353_1.`1User`ORDER BY isEmployer DESC";
```
Comment (TaillessSheep) [inline admin_user_reports.php]: maybe remove the _oyc353_1._ if it is not necessary

File: admin_user_reports.php (hunk 1 above)
Comment (TaillessSheep) [inline admin_user_reports.php]: defined variable: deactivate_err

**Review by TaillessSheep — CHANGES_REQUESTED** [review]

File: admin_user_reports.php (hunk 2 above)
Comment (AxelBogos) [reply admin_user_reports.php]: Done

File: admin_user_reports.php (hunk 4 above)
Comment (AxelBogos) [reply admin_user_reports.php]: Done

File: admin_user_reports.php (hunk 1 above)
Comment (AxelBogos) [reply admin_user_reports.php]: Done

**Review by TaillessSheep — APPROVED** [review]

**Pull request merged**